client/src/components/PageExplorer/PageExplorer.scss

```
@@ -44,8 +44,8 @@ $explorer-header-horizontal-padding: 10px;
   display: grid;
   grid-template-columns: 1fr auto;
   align-items: center;
-  margin-inline-start: $sidebar-toggle-spacing * 2 + $sidebar-toggle-size;
-  height: $sidebar-toggle-spacing * 2 + $sidebar-toggle-size;
+  margin-inline-start: 50px;
+  height: 48px;
```

We should almost always use variables instead of hard-coded values.
Turns out that this should exactly match the actual toggle size (50px) and we have a variable for that.

```suggestion
  margin-inline-start: $mobile-nav-indent;
  height: $mobile-nav-indent;
```